mdoc-sbt/src/main/scala/mdoc/DocusaurusPlugin.scala

```diff
@@ -58,7 +58,7 @@ object DocusaurusPlugin extends AutoPlugin {
        |  mkdir -p $HOME/.ssh
        |  ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
        |  git config --global user.name "Docusaurus bot"
-       |  git config --global user.email "$TRAVIS_BUILD_NUMBER@$TRAVIS_COMMIT"
+       |  git config --global user.email "${TRAVIS_BUILD_NUMBER:-${CIRCLE_BUILD_NUM}}@${TRAVIS_COMMIT:-${CIRCLE_SHA1}}"
```

Comment:
I think it’s best to replace the Travis specific email with a static value like “mdoc” or a dummy email.